web_accounts_list.json

```diff
@@ -2994,6 +2994,50 @@
          "known_accounts" : ["jdheyes", "healthranger"],
          "category" : "political",
          "valid" : true
+      },
+      {
+         "name" : "SaraCarterShow",
+         "check_uri" : "https://saraacarter.com/author/{account}/",
+         "account_existence_code" : "200",
+         "account_existence_string" : ", Author at Sara A. Carter : Sara A. Carter</title>",
+         "account_missing_string" : "We're sorry, but that page could not be found.",
+         "account_missing_code" : "404",
+         "known_accounts" : ["douglasbraff", "annaliese"],
+         "category" : "political",
+         "valid" : true
+      },
+      {
+         "name" : "Bitchute",
+         "check_uri" : "https://www.bitchute.com/channel/{account}/",
+         "account_existence_code" : "200",
+         "account_existence_string" : "subscribers",
+         "account_missing_string" : "404 - Page not found",
+         "account_missing_code" : "404",
+         "known_accounts" : ["simon_parkes", "americafloats", "daindor"],
+         "category" : "political",
+         "valid" : true
+      },
+      {
+         "name" : "thegatewaypundit",
+         "check_uri" : "https://www.thegatewaypundit.com/author/{account}/",
+         "account_existence_code" : "200",
+         "account_existence_string" : "summary",
+         "account_missing_string" : "Not found, error 404",
+         "account_missing_code" : "404",
+         "known_accounts" : ["patti", "joehoft", "daindor"],
+         "category" : "political",
+         "valid" : true
+      },
+      {
+         "name" : "Wimkin",

```

Comment:
the john account has a different response. please adjust the account_existence_string